docs/_articles/en/einstein/einstein-overview.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Einstein for Developers Overview
+lang: en
+---
+
+Einstein for Developers is an AI-powered developer tool that is available as an easy-to-install VS Code extension built using CodeGen, Salesforce’s secure, custom AI model. Use the tool to enhance developer productivity by quickly generating code suggestions using natural language instructions. The extension is available in the [VS Code](https://marketplace.visualstudio.com/vscode) and [Open VSX](https://open-vsx.org/) marketplaces.
+
+Einstein for Developers assists you throughout the Salesforce development process with expertise learned from anonymized code patterns. The goal of our suite of AI-powered developer tools are to increase productivity and provide helpful assistance for complex coding tasks. We enforce development best practices with our code generation and just-in-time code completion features. AI-assisted tooling also makes it easier for new developers to onboard to the Salesforce Platform.
+
+## Current Capabilities
+
+This Einstein for Developers(beta) focuses on natural language to Apex code generation. This feature, used along with IntelliSense, makes Apex development tooling in Visual Studio Code even richer. Familiarity with Visual Studio Code is assumed.
```

Comment:
(I'm assuming this is the case but just curious :) ) are we required to keep the `(beta)` suffix whenever we refer to the product?

Reply:
I got some info on this. We need to add a legal disclaimer after the title, and then we can skip the "beta" suffix. I'll make the change.